pom.xml

```diff
@@ -1418,7 +1426,7 @@ under the License.
 										<!-- versions for certain build tools are enforced to match the CI setup -->
 										<!-- the rules below should stay in sync with Flink Release wiki documentation and the CI scripts -->
 										<requireJavaVersion>
-											<version>[11.0.0,11.1.0)</version>
+											<version>[17.*)</version>
```

Comment:
```suggestion
											<version>[${target.java.version}.*)</version>
```

Reply:
same reason about configurable java version for enforcer
as mentioned at 
https://github.com/apache/flink/pull/25898/files#r1907847458